plutus-core/plutus-core/src/PlutusCore/Compiler/Erase.hs

```diff
@@ -1,10 +1,13 @@
 module PlutusCore.Compiler.Erase (eraseTerm, eraseProgram) where
 
 import PlutusCore.Core
+import PlutusCore.Name
 import UntypedPlutusCore.Core qualified as UPLC
 
 -- | Erase a Typed Plutus Core term to its untyped counterpart.
-eraseTerm :: Term tyname name uni fun ann -> UPLC.Term name uni fun ann
+eraseTerm :: HasUnique name TermUnique
```

Comment:
Add a short comment please. And yeah, I think long-term we should fix debruijnification and undo this change. It's fine for now, though.